tests/test_search.py

```diff
@@ -0,0 +1,117 @@
+import os
+
+import unittest
+from unittest.mock import patch
+from flask.testing import FlaskClient
+from iiify.app import app
+from iiify import search
+import json
+import requests
+
+def mockResponse(fixture):
+    class MockResponse:
+        status_code = 200
+        def json(self):
+            with open(fixture, "r") as file:
+                return json.load(file)
+
+        def raise_for_status(self):
+            return        
+
+    return MockResponse()
+
+class TestSearch(unittest.TestCase):
```

Comment:
We really don't have a test for anything related to `iiif_search()`.  Should we try to mock / patch / test it as is, or break it into smaller chunks that could get a test?  To be honest, I don't care except for showing that `re` and `bs4` will work the same.

Reply:
As discussed I think this is already done in the existing tests.